tests/MongoDB.Driver.Tests/PipelineStageDefinitionBuilderTests.cs

```diff
@@ -437,5 +518,49 @@ private RenderedPipelineStageDefinition<ChangeStreamDocument<BsonDocument>> Rend
             var serializer = registry.GetSerializer<TInput>();
             return stage.Render(serializer, registry);
         }
+
+        // nested types
+        private class A
+        {
+            public X From { get; set; }
+            public X To { get; set; }
+        }
```

Comment:
Similar changes as in `AggregateGraphLookupEnumerableFromOrToTests.cs` for these 6 test classes.